src/main/java/app/controller/linAlg/Angle.java

```diff
@@ -11,7 +11,7 @@ public static boolean angleInRange(double angle, double upper, double lower)
 
         if(a < b)
             return a <= N && N <= b;
-        throw new RuntimeException("Upper bound needs to be strictly above the lower bound");
+        return a <= N || N <= b;
```

Comment:
This evaluation doesn't make logical sense.  The AND statement is the same as the OR statement, so the OR by itself will always run ... ?

Reply:
I am actually not sure what this is for, someone else (I think Laura, but not sure) told me to put it in there. But it indeed doesn't make sense to seperate it.